scripts/jestIntegration.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "integration",
```

Comment:
It should probably be named `puppeteer` to match `node` & `jsdom` (name the tool rather than the type of test).